script.js

```diff
@@ -0,0 +1,155 @@
+let currentTasks = [];
+let completedTasks = [];
+const maxCompletedTasksToShow = 5;
+
+const taskForm = document.querySelector(".todo");
+const taskList = document.querySelector(".todo-list");
+const completedList = document.querySelector(".solved-list");
+const submitButton = document.querySelector("button");
+
+const resetInputText = () => {
+  const taskInput = document.querySelector(".todo > input");
+  taskInput.value = "";
+};
+
+const addTaskToList = (text) => {
+  if (!currentTasks.includes(text)) {
+    const listItem = document.createElement("li");
+    listItem.innerHTML = text;
+    currentTasks.push(text);
+    taskList.appendChild(listItem);
+  }
+};
+
+const addCompletedTaskToList = (text) => {
+  if (!completedTasks.includes(text)) {
+    const listItem = document.createElement("li");
+    const textElement = document.createElement("p");
+    textElement.innerHTML = text;
+    const deleteImg = document.createElement("img");
+    deleteImg.src = "./images/delete.png";
+    const restoreImg = document.createElement("img");
+    restoreImg.src = "./images/restore.png";
+    listItem.appendChild(deleteImg);
+    listItem.appendChild(restoreImg);
+    listItem.appendChild(textElement);
```

Comment:
저도 똑같이 피드백 받은 부분인데 append() 메서드를 사용해서 child를 한번에 append하는 방식을 사용해도 좋을 것 같아요~!
```suggestion
    listItem.append(deleteImg, restoreImg, textElement);
```